api/server_test.go

```diff
@@ -38,6 +38,12 @@ func TestGetHttpServer(t *testing.T) {
 		expectedStatusCode int
 		failAuth           bool
 	}{
+		{
+			name:               "Healthcheck",
+			url:                "http://relay-meter.pokt.network/",
```

Comment:
not too comfortable having this explicitly there, would rather have an env

Reply:
This is already explicit on all test cases, we can add it as a betterment for later